src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -770,3 +774,9 @@ def _parse_drs_uri(self,
                 return str(file_id.path).strip('/')
         else:
             return None
+
+
+def _disallow_joiner(content: str) -> None:
```

Comment:
Placing this function here is going to cause problems if/when we add additional plugins that need to access it. I know we aren't enforcing the absence of the joiner on the DSS or Canned plugins right now, but my understanding is that this is due to a technical limitation. The AnVIL plugin I'm currently working on will definitely need access to this eventually. Consider moving this function to `src/azul/__init__.py` and making it public.